pkg/minikube/assets/addons.go

```diff
@@ -142,6 +142,18 @@ var Addons = map[string]*Addon{
 			"ingress-svc.yaml",
 			"0640"),
 	}, false, "ingress"),
+	"registry": NewAddon([]*MemoryAsset{
```

Comment:
The addons feature has a dependency on the `minikube config` feature, so these need an additional entry in the list here

https://github.com/kubernetes/minikube/blob/master/cmd/minikube/cmd/config/config.go#L45-L174

Then you can enable/disable by `minikube addons enable registry`